lib/json/static.nit

```diff
@@ -152,56 +146,36 @@ redef class FlatText
 	end
 end
 
-redef class Int
-	super Jsonable
-end
-
-redef class Float
-	super Jsonable
-end
-
-redef class Bool
-	super Jsonable
-end
-
 # A map that can be translated into a JSON object.
-interface JsonMapRead[K: String, V: nullable Jsonable]
+interface JsonMapRead[K: String, V: nullable Serializable]
 	super MapRead[K, V]
-	super Jsonable
+	super Serializable
 end
 
 # A JSON Object.
 class JsonObject
```

Comment:
Do you have a usage for this class other than as a shorter alternative to `HashMap[String, nullable Serializable]`?

It's a good reason to keep it, but if it's only as a shortcut, @privat could be interested in replacing it by a virtual type (of the same name) on `Object`.